charts/astronomer/templates/ingress.yaml

```diff
@@ -42,24 +42,15 @@ spec:
       paths:
         - path: /
           backend:
-            serviceName: {{ .Release.Name }}-orbit
-            servicePort: orbit-http
+            serviceName: {{ .Release.Name }}-astro-ui
+            servicePort: astro-ui-http
   - host: app.{{ .Values.global.baseDomain }}
     http:
       paths:
         - path: /
           backend:
-            serviceName: {{ .Release.Name }}-orbit
-            servicePort: orbit-http
-{{- if .Values.norbit.enabled }}
-  - host: beta.{{ .Values.global.baseDomain }}
-    http:
-      paths:
-        - path: /
-          backend:
-            serviceName: {{ .Release.Name }}-norbit
-            servicePort: norbit-http
```

Comment:
I prefer if we keep the "beta." functionality around. I hope we can enable the front end team to push to this unconstrained by a release process.

Reply:
for example, the service could have pull policy always and point to astro-ui:latest. And somehow gets triggered to update when a new one is published

Reply:
perhaps we just put it back in with a different change, that may be outside the scope of this change.